### PR TITLE
[dev] - Product Backlog Item AB#70839: Add appium work for automation - add element id for accessibility- lower prompts elements and button visibility in appium android

### PR DESCRIPTION
**Pull request by pakkibokati2012**
**Summary**

- >Button component's accessibility id now visible in Appium inspector in android
- >LowerPrompt components text and buttons now inspectable as well in Appium in android

** Closes** #90

### CODE REVIEW

**Review by ajaybushel — APPROVED** [review]
Looks good to me. However, couldn't we use `test id` instead of adding `accessibility label`s?

**Comment by pakkibokati2012** [comment]
> 



> Looks good to me. However, couldn't we use `test id` instead of adding `accessibility label`s?
Screen Shot 2022-04-17 at 14 02 56
The testing automation team has explicitly asked for the` accessiblity id` field to show up on appium inspector. This cannot be achieved without setting the` accessiblity label` prop.

**Comment by ajaybushel** [comment]
Understood. Thanks @pakkibokati2012

File: src/modules/button/button.tsx [hunk 1]
```diff
@@ -208,6 +209,7 @@ const Button: React.FC<IProps> = React.memo(
         {/* Label */}
         <StyledLabel
           $font={font}
+          accessibilityLabel="button-label"
```
Comment (mdjacobs) [inline src/modules/button/button.tsx:212]: My only concern is that if there are say 5 buttons on the page, automation will not know which one to pick.

**Review by mdjacobs — CHANGES_REQUESTED** [review]
If we can make some changes as best as possible to include some parameters to allow automation to target these elements in both buttons and lower prompts.

**Comment by mdjacobs** [comment]
@pakkibokati2012 can you do some adjustments to help target these elements better?

**Review by raoufr — APPROVED** [review]
Looking good to me

**Pull request merged**